protocol.html

```diff
@@ -500,6 +500,12 @@ <h2 property="schema:name">Protocol</h2>
                 subscription requirements of the notification protocol.
               </p>
 
+              <p id="json-ld-format">This specification uses JSON-LD [<cite><a class="bibref" href="#bib-json-ld11">JSON-LD11</a></cite>] as the preferred data format, and <code>https://www.w3.org/ns/solid/notification/v1</code> as the URI for the JSON-LD Context and as a value of the <code>profile</code> parameter in content negotiation.</p>
+
+              <p id="json-ld-context">JSON-LD representations employing the JSON-LD context MUST ensure that the URI <code>https://www.w3.org/ns/solid/notification/v1</code> is the only value or the terms defined within the context are not overridden by another context.</p>
```

Comment:
(just noting that the MUST here is not on a client or server, but on a non-actor)

Reply:
Good note.

https://github.com/solid/notifications/pull/120 propose to identify client, resource server, and notification server as the classes of products for conforming implementations. I didn't want to split the statement into three conformance clauses until we had rough consensus.

Alternatively, "content" (of type, meaning, and format as defined in the specification) could be introduced as a product class. The current text is closer to expressing the conformance clause with "content".

Not sure which approach would be preferable - it depends on how we categorise the other requirements. Can we come back to this?

Will look into including JSON-LD concepts/behaviours here - overridden is fine but perhaps should mentioned "protected"?

Reply:
So long as we refer to JSON-LD 1.1 and so long as this context uses the `@protected` parameter in its definition (it does), then the second sentence here is unnecessary.

Reply:
Removing `#json-ld-context` is okay with me. @RubenVerborgh ?

However, `#json-ld-format` may need to be updated or another paragraph needs be introduced to express that `#expanded-document-form` (using the vocabulary URIs directly) would also conform. Or would you consider that to be the case and not conflicting with the way `#json-ld-format` is currently expressed?

Reply:
I am okay with that (and it's good input for security considerations / RDF serializations in general).

Reply:
Resolving thread with changes in https://github.com/solid/notifications/pull/122/commits/0d4afa35455909e5195a990f0a2883c0b834e542 , https://github.com/solid/notifications/pull/122/commits/4a0b6be9f1cc889c80ae11b86e67c0f192a20751